<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Orwell</class>
 <widget class="QMainWindow" name="Orwell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Feedback Test Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Version:</string>
        </property>
        <property name="buddy">
         <cstring>version</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="version"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Starts:</string>
        </property>
        <property name="buddy">
         <cstring>startCount</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="startCount"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Runtime:</string>
        </property>
        <property name="buddy">
         <cstring>runtime</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="runtime">
        <property name="suffix">
         <string>secs</string>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="singleStep">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Completed Surveys:</string>
        </property>
        <property name="buddy">
         <cstring>surveys</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="surveys">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Survey Interval:</string>
        </property>
        <property name="buddy">
         <cstring>surveyInterval</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="surveyInterval">
        <property name="specialValueText">
         <string>Never</string>
        </property>
        <property name="suffix">
         <string>days</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>90</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="overrideButton">
        <property name="text">
         <string>Override Stats</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="submitButton">
        <property name="text">
         <string>Submit Manually</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDial" name="dial">
        <property name="maximum">
         <number>11</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Property Source Test:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionContribute"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionContribute">
   <property name="text">
    <string>&amp;Contribute...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
